### PR TITLE
Clean up admin page

### PR DESCRIPTION
**Pull request by TomAbel**
Ready for review

### CODE REVIEW

File: lang/smaily-for-wp-et.po [hunk 1]
```diff
@@ -1,9 +1,9 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Smaily for WP\n"
-"POT-Creation-Date: 2021-02-03 12:58+0200\n"
-"PO-Revision-Date: 2021-02-03 12:58+0200\n"
-"Last-Translator: Smaily <info@smaily.com>\n"
+"POT-Creation-Date: 2021-02-18 11:11+0200\n"
+"PO-Revision-Date: 2021-02-18 11:11+0200\n"
+"Last-Translator: Tom <tom@example.com>\n"
```
Comment (kaittodesk) [inline lang/smaily-for-wp-et.po:6]: Make sure this is `Smaily <info@smaily.com>`.

File: lang/smaily-for-wp.pot [hunk 2]
```diff
@@ -2,10 +2,10 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Smaily for WP\n"
-"POT-Creation-Date: 2021-02-03 12:57+0200\n"
+"POT-Creation-Date: 2021-02-18 11:10+0200\n"
 "PO-Revision-Date: 2021-01-05 20:12+0200\n"
 "Last-Translator: Smaily <info@smaily.com>\n"
-"Language-Team: Smaily <info@smaily.com>\n"
+"Language-Team: Smaily <language@example.com>\n"
```
Comment (kaittodesk) [inline lang/smaily-for-wp.pot:8]: If you'd like the `Language-Team` value to be undetermined then change the name to something like `Language Team` also.

**Review by kaittodesk — CHANGES_REQUESTED** [review]

File: lang/smaily-for-wp-et.po (hunk 1 above)
Comment (TomAbel) [reply lang/smaily-for-wp-et.po]: Yep. I didn't want to change this. c8a47da

File: lang/smaily-for-wp.pot (hunk 2 above)
Comment (TomAbel) [reply lang/smaily-for-wp.pot]: Changed it back c8a47da

**Comment by kaittodesk** [comment]
One more thing. Is the `lang/smaily-for-wp-et.mo` file supposed to be empty?

**Comment by TomAbel** [comment]
I don't know why it shows as empty. It actually is filled if you look at the source https://github.com/sendsmaily/sendsmaily-wordpress-plugin/blob/c8a47da531db4ad19a8513f85ed44aecd2843b4b/lang/smaily-for-wp-et.mo

**Comment by kaittodesk** [comment]
Oh, OK. Thanks for letting me know.

**Review by kaittodesk — APPROVED** [review]

**Pull request merged**